web/styles/popovers.css

```diff
@@ -402,7 +402,9 @@ ul {
 
 .popover-avatar {
     height: 240px;
-    width: 240px;
+    /* The popover container has width set to 240px and we reduce the width
+    by 2px (1px for border on each side) to get width of the avatar. */
+    width: 238px;
```

Comment:
I think it'd be better to change the popover width to 242px, so we don't distort the image by displaying it at a non-square size. Arguably we should be doing 252px width for the popover, such that the avatar is 250x250, since `MEDIUM_AVATAR_SIZE = 500` means the actual images we're showing are 500x500 square assets?

@karlstolley @terpimost for thoughts on that second suggestion.

Reply:
Yes. Let's keep clear a relationship between the asset size and its appearance in the browser.